src/components/Footer.svelte

```diff
@@ -6,7 +6,7 @@
   const currentYear = new Date().getFullYear();
 </script>
 
-<footer class="flex justify-between flex-wrap p-6">
+<footer class="flex justify-between flex-wrap p-6 pb-0">
```

Comment:
`px-6 pt-6` would be better than adding a padding everywhere and then remove the bottom one